### PR TITLE
docs: display_status warning in git segment doc + issue template updated

### PR DESCRIPTION
**Pull request by lnu**
### Prerequisites

- [x] I have read and understand the `CONTRIBUTING` guide
- [x] The commit message follows the [conventional commits][cc] guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

### Description

Doc updated for `display_status` change to false by default + issue template updated with optional git and posh-git version

### Tips

If you're not comfortable working with git, we're working a [guide][docs] to help you out.
Oh my Posh advises [GitKraken][kraken] as your preferred cross platfrom git GUI powertool.

[cc]: https://www.conventionalcommits.org/en/v1.0.0/#summary
[docs]: https://ohmyposh.dev/docs/contributing_git
[kraken]: https://www.gitkraken.com/invite/nQmDPR9D

### CODE REVIEW

File: docs/docs/segment-git.mdx [hunk 1]
```diff
@@ -19,6 +19,13 @@ PowerShell offers support for the `posh-git` module for autocompletion, but it i
 To enable this, set `$env:POSH_GIT_ENABLED = $true` in your `$PROFILE`.
 :::
 
+:::warning
+Starting from version 3.152.0, `display_status` is disabled by default.
+It improves performance but reduces the quantity of information.
+Don't forget to enable it in your theme if needed.
+An alternative is to use the [Posh-Git segment][poshgit]
```
Comment (JanDeDobbeleer) [inline docs/docs/segment-git.mdx:25]: posh-git will only work in Powershell, we might want to highlight that too here

Comment (lnu) [reply docs/docs/segment-git.mdx:25]: I thought about it too, but it's the first information in the posh-git segment :).
I can add it here too.

Comment (lnu) [reply docs/docs/segment-git.mdx:25]: image

Comment (lnu) [reply docs/docs/segment-git.mdx:25]: image

**Comment by JanDeDobbeleer** [comment]
Let's merge it 😀

**Pull request merged**